ports/stm32/boards/Passport/modules/flows/envoy_setup_flow.py

```diff
@@ -20,37 +20,24 @@ def __init__(self):
         super().__init__(initial_state=self.check_if_envoy_installed, settings_key='envoy_setup', name='EnvoySetupFlow')
```

Comment:
Shouldn't we set the initial state to `show_envoy_app_url()` here if we are deprecating `check_if_envoy_installed()`?